core/templates/pages/exploration-editor-page/services/change-list.service.ts

```diff
@@ -30,6 +30,55 @@ import { LoggerService } from 'services/contextual/logger.service';
 import { ExplorationChange } from 'domain/exploration/exploration-draft.model';
 import { WindowRef } from 'services/contextual/window-ref.service';
 import { InternetConnectivityService } from 'services/internet-connectivity.service';
+import { SubtitledHtml, SubtitledHtmlBackendDict } from 'domain/exploration/subtitled-html.model';
+import { ParamChange, ParamChangeBackendDict } from 'domain/exploration/ParamChangeObjectFactory';
+import { InteractionCustomizationArgs, InteractionCustomizationArgsBackendDict } from 'interactions/customization-args-defs';
+import { AnswerGroup, AnswerGroupBackendDict } from 'domain/exploration/AnswerGroupObjectFactory';
+import { Hint, HintBackendDict } from 'domain/exploration/HintObjectFactory';
+import { Outcome, OutcomeBackendDict } from 'domain/exploration/OutcomeObjectFactory';
+import { RecordedVoiceOverBackendDict, RecordedVoiceovers } from 'domain/exploration/recorded-voiceovers.model';
+
+export type StatePropertyValues = (
+  SubtitledHtml |
+  ParamChange[] |
+  string |
+  InteractionCustomizationArgsBackendDict |
+  AnswerGroup[] |
+  Hint[] |
+  Outcome |
+  RecordedVoiceovers |
+  boolean
```

Comment:
Can this be in sorted order here and elsewhere?

Reply:
Done.